website/websocket/src/main.ts

```diff
@@ -340,6 +351,15 @@ io.on('connection', (socket) => {
     socket.on('disconnect', () => {
         connectedUsers.delete(user.id);
         io.emit('users_count', connectedUsers.size);
+        const ip = getClientIp(socket);
```

Comment:
Why are you re-fetching the user's ip if you already have it on line 199? Other than that LGTM

Reply:
```suggestion

```